ui/src/app/views/workspace/component.html

```diff
@@ -0,0 +1,61 @@
+<h2>AoU Research Portal Workspace</h2>
+<div *ngIf="user else loggedout">
+  <p class="description-text">Default workspace for July, 2017 demo</p>
+  <div class="area-block">
+    <h3>CDR Version</h3>
+    <div class="indented">
+      <!-- TODO: Add CDR Version Info - blrubenstein -->
```

Comment:
No need to fix, but a TODO format I've seen used commonly is: `TODO(blrubenstein): Add CDR Version Info`